### PR TITLE
[Appconfig] Fix content validation issues for github.io

### PR DESCRIPTION
**Pull request by YalinLi0312**
For issue https://github.com/Azure/azure-sdk-for-python/issues/37143

### CODE REVIEW

File: sdk/appconfiguration/azure-appconfiguration/azure/appconfiguration/_azure_appconfiguration_client.py [hunk 1]
```diff
@@ -109,7 +109,8 @@ def from_connection_string(cls, connection_string: str, **kwargs: Any) -> "Azure
 
         .. code-block:: python
 
-            from azure.appconfiguration import AzureAppConfigurationClient
+            from azure.appconfiguration import
```
Comment (kristapratico) [inline sdk/appconfiguration/azure-appconfiguration/azure/appconfiguration/_azure_appconfiguration_client.py:112]: we lost the client in the import

File: sdk/appconfiguration/azure-appconfiguration/azure/appconfiguration/_azure_appconfiguration_client.py [hunk 2]
```diff
@@ -212,7 +213,7 @@ def list_configuration_settings(
         """
 
     @distributed_trace
-    def list_configuration_settings(self, *args, **kwargs) -> ItemPaged[ConfigurationSetting]:
+    def list_configuration_settings(self, *args: Optional[str], **kwargs: Any) -> ItemPaged[ConfigurationSetting]:
```
Comment (kristapratico) [inline sdk/appconfiguration/azure-appconfiguration/azure/appconfiguration/_azure_appconfiguration_client.py:216]: would a user actually pass None for args?

File: sdk/appconfiguration/azure-appconfiguration/azure/appconfiguration/_azure_appconfiguration_client.py (hunk 1 above)
Comment (YalinLi0312) [reply sdk/appconfiguration/azure-appconfiguration/azure/appconfiguration/_azure_appconfiguration_client.py:112]: Oh no, that's removed by accident.

File: sdk/appconfiguration/azure-appconfiguration/azure/appconfiguration/_azure_appconfiguration_client.py (hunk 2 above)
Comment (YalinLi0312) [reply sdk/appconfiguration/azure-appconfiguration/azure/appconfiguration/_azure_appconfiguration_client.py:216]: That's allowed as the args are optional, but I don't think users will pass None for them.

**Review by kristapratico — APPROVED** [review]

**Pull request merged**